gui/widget.cpp

```diff
@@ -403,13 +415,14 @@ void ButtonWidget::setLabel(const Common::String &label) {
 }
 
 void ButtonWidget::setLowresLabel(const Common::U32String &label) {
-	_lowresLabel = label;
+	_lowresLabel = cleanupHotkey(label);
 }
 
 const Common::U32String &ButtonWidget::getLabel() {
 	bool useLowres = false;
 	if (!_lowresLabel.empty())
-		useLowres = g_gui.theme()->getStringWidth(_label) > _w;
+		useLowres = g_system->getOverlayWidth() <= 320;
```

Comment:
use screen width instead of button width, because we don't have automated testing for text translations fitting in the GUI so we don't want to hide problems with really long translations